src/frontend/sider/Fagsak/Behandling/Sider/Behandlingsresultat/EndretUtbetaling/komponenter/AvtaletidspunktDeltBostedDatovelger.test.tsx

```diff
@@ -0,0 +1,233 @@
+import type { PropsWithChildren, ReactNode } from 'react';
+import React from 'react';
+
+import { type DefaultValues, FormProvider, useForm } from 'react-hook-form';
+import { describe, expect, test } from 'vitest';
+
+import { Button } from '@navikt/ds-react';
+
+import AvtaletidspunktDeltBostedDatovelger from './AvtaletidspunktDeltBostedDatovelger';
+import { render } from '../../../../../../../testutils/testrender';
+import { IEndretUtbetalingAndelÅrsak } from '../../../../../../../typer/utbetalingAndel';
+import { EndretUtbetalingAndelFeltnavn, type EndretUtbetalingAndelFormValues } from '../useEndretUtbetalingAndelRHF';
+
+const DEFAULT_VALUES: DefaultValues<EndretUtbetalingAndelFormValues> = {
+    [EndretUtbetalingAndelFeltnavn.ÅRSAK]: '',
+    [EndretUtbetalingAndelFeltnavn.AVTALETIDSPUNKT_DELT_BOSTED]: undefined,
```

Comment:
```suggestion
    [EndretUtbetalingAndelFeltnavn.AVTALETIDSPUNKT_DELT_BOSTED]: null,
```

> You should avoid providing undefined as a default value, as it conflicts with the default state of a controlled component.

Se under "Rules" her https://react-hook-form.com/docs/useform 

Ser det gjelder flere steder i denne filen så bare sjekk om hvor du bruker `undefined`